### ISSUE TITLE
Japanese localization is incomplete (some items are even blank)

### ISSUE THREAD

**Issue by RadarNyan** [XITRIX/Moonlight-Switch#178]
I'd like to help to complete the Japanese localization, but the link to the POEditor website in the project's README section just leads me to a page saying, "There are no languages to translate in this localization project."

**Comment by XITRIX** [comment]
POEditor was not used for so long that it archived the project and revoked open source license from it 🥲

I unarchived it, but I need to wait for reapproval of open source license to allow you to edit it, it should happen in 2-3 days

Thanks for your interest in translating the app

**Comment by RadarNyan** [comment]
It's been a week, and I'm still getting the message, "This project can't accept new contributors at this time. Please try again later." on POEditor.

I've been translating ```resources/i18n/en-US/main.json```, but the structure of the file isn't very clear where the text actually belongs (well, I guess it's a common issue), so I had to compare it to the actual app running on the switch to make changes, which takes some time to finish.

For some texts, I'm not sure how or if I should translate them since I can't find them in the app, and for those I've found in the app, I'm not sure if it's correct to "translate" the word into something completely different. For example, I've changed```"common": { "close": "Close"``` into ```"common": { "close": "OK"``` since the only place I found this text is the warning message when launching the app in Applet Mode and following the phrasing of Nintendo's official system UI, the word should be "OK" for a message like this, but it feels wrong to "translate" the word "close" to "OK".

**As of v1.2.0, there's a slider with a default value of "x1.0" in the settings page above the debug section, and this slider has no text indication. What is this slider for, and how would I assign text to it?**

I'll just attach the JSON file here if the POEditor issue is not resolved when I finish the translation.

**Comment by RadarNyan** [comment]
There are some more problems I've found trying to translate:

1. Sharing some texts under the ```common``` group causes problems; for example, the word "cancel" could be translated into "キャンセル (cancel)" or "もどる (go back)" depending on where they appear. Surely, it's just a small detail that doesn't hurt that much, but **I think it's a better idea to just not share the text at all.** It would provide a clearer structure - all texts that belong to a single page remain in its own group.

2. Some translations can't be done correctly since the order of the elements is different across languages. For example, when terminating an app, the message in English says "Terminate [App Name]", but in Japanese, the correct order would be "[App Name] is to be terminated". Again, it's just a small detail, but it makes the translation feel strange to native speakers.

**Comment by RadarNyan** [comment]
I've completed a rough translation of the main.json file. Since POEditor is still not working for me, I just uploaded it to GitHub: https://github.com/RadarNyan/Moonlight-Switch/blob/i18n/resources/i18n/ja/main.json

Some texts are not translated because I couldn't find them in the app, so I can not confirm how they're used.

Also, about the slider with no text on the settings page, I believe it's this one: ```"mouse_speed": "Mouse acceleration"```. However, there's a problem: the key says "mouse_speed" yet the value says "Mouse acceleration". Which one is it? Mouse speed and acceleration are two completely different things.
